### PR TITLE
Add links to description metadata

### PR DESCRIPTION
**Pull request by arturo-seijas**

### CODE REVIEW

**Review by varshigupta12 — COMMENTED** [review]
LGTM

**Review by varshigupta12 — DISMISSED** [review]

**Review by weiiwang01 — DISMISSED** [review]

**Review by nrobinaubertin — DISMISSED** [review]

**Review by merkata — DISMISSED** [review]

File: localstack-installation.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 pip install pip --upgrade
 pip install pyopenssl --upgrade
-pip install localstack # install LocalStack cli
+pip install 'localstack>=1.4.0,<2.0.0' # install LocalStack cli
 docker pull localstack/localstack # Make sure to pull the latest version of the image
```
Comment (weiiwang01) [inline localstack-installation.sh:6]: Actually, I was considering pinning the docker image of localstack. For now, it's using the `latest` version of the localstack docker image, which is a nightly build.

**Review by merkata — APPROVED** [review]

File: localstack-installation.sh (hunk 1 above)
Comment (arturo-seijas) [reply localstack-installation.sh:6]: It doesn't seem to be problematic. We can pin it in a follow up PR if that works for you

**Review by amandahla — APPROVED** [review]
LGTM

**Pull request merged**